### DISCUSSION TITLE
Why use OxyPlot?

### DISCUSSION THREAD

**Discussion by damonpkuml** [silahian/VisualHFT#39] [General]
Hello, thank you for your contributions to the open-source community! May I ask why you chose to use OxyPlot instead of similar libraries like ScottPlot? Is it due to performance or other considerations? I hope we can have more discussions on this topic.

**Comment by silahian** [comment]
Thanks for your input.
We have tried many others and their performance was terrible.
OxyPlot is not the best, but it had good enough performance for what we needed at this stage.

The problem is that we do not need to plot a huge amount of points, we are plotting 1-5k points per chart.
Tha main problem in our use case, is that those points needs to be refresh in under 300 microseconds (for real-time usage)


My idea is to port all these chart for a much more basic one with much better performance.
GLGraph looks a great alternative for this, but needs to have some tweaks...

Anyone looking to collaborate, that would be a great way to do so :)
